third-party/stb_image.h

```diff
@@ -1644,13 +1644,13 @@ stbi_inline static int stbi__extend_receive(stbi__jpeg *j, int n)
    int sgn;
    if (j->code_bits < n) stbi__grow_buffer_unsafe(j);
 
-   sgn = (stbi__int32)j->code_buffer >> 31; // sign bit is always in MSB
+   sgn = j->code_buffer >> 31; // sign bit always in MSB; 0 if MSB clear (positive), 1 if MSB set (negative)
```

Comment:
For third party we should first check and priorities version updates, and license notice for modification.

Reply:
this is official fix, i provided link to original repo

Reply:
So lets consider taking the latest version:
![image](https://user-images.githubusercontent.com/64067618/216965452-d21fba14-2167-4a4f-8b20-942fd3a4234c.png)

Just need to see camera images looks OK for windows / linux

Reply:
https://github.com/nothings/stb/blob/master/stb_image.h

Reply:
We will handle stb image on a dedicated PR